plugins/org.python.pydev.core/src/org/python/pydev/core/UnpackInfo.java

```diff
@@ -11,20 +11,26 @@
     private int unpackTuple = -1;
     private boolean unpackFor = false;
     private boolean unpackBackwards;
+    private boolean unpackWith;
```

Comment:
I can't see any use-case where the `getUnpackWith()` would be used (so, unless it actually does have some usage, I think it should be removed, as well as the related constructor -- we can add it back when needed).